godotproject/Menus/InGameMenu/QuitGameButton.gd

```diff
@@ -1,4 +1,4 @@
 extends Button
 
 func _on_QuitGameButton_pressed():
-	get_tree().quit()
+	Global.fade_to_scene('res://Menus/MainMenu/MainMenu.tscn')
```

Comment:
When we select "quit" from the in-game menu, don't close the app; instead go back to the main menu. This makes sense for the arcade machine since the user shouldn't be able to close the game.

Reply:
Good stuff, this is definitely the right call